src/content.js

```diff
@@ -301,6 +325,18 @@ export function nuke (host, range, tagName) {
   })
 }
 
+// Unwrap all tags this range is affected by.
+// Can also affect content outside of the range.
+export function nukeElem (host, range, node) {
+  getTags(host, range).forEach((elem) => {
+    if (elem.nodeName !== 'BR' && (!node ||
+        (elem.nodeName === node.nodeName.toUpperCase() &&
```

Comment:
same here: why is `toUpperCase()` only used once (this also goes for the line directly above: why not there?)

Reply:
because I just adapted the nuke () function. But I changed it.